src/panels/config/server_control/ha-config-section-server-control.js

```diff
@@ -0,0 +1,222 @@
+                  <template is="dom-if" if="[[validating]]">
+                    <paper-spinner active=""></paper-spinner>
+                  </template>
+                </div>
+              </template>
+              <template is="dom-if" if="[[validateLog]]">
+                <div class="config-invalid">
+                  <span class="text">
+                    [[localize('ui.panel.config.server_control.section.validation.invalid')]]
+                  </span>
+                  <mwc-button raised="" on-click="validateConfig">
+                    [[localize('ui.panel.config.server_control.section.validation.check_config')]]
+                  </mwc-button>
+                </div>
+                <div id="configLog" class="validate-log">[[validateLog]]</div>
+              </template>
+            </div>
+          </ha-card>
+
+          <ha-card
+            header="[[localize('ui.panel.config.server_control.section.reloading.heading')]]"
+          >
+            <div class="card-content">
+              [[localize('ui.panel.config.server_control.section.reloading.introduction')]]
+            </div>
+            <div class="card-actions">
+              <ha-call-service-button
+                hass="[[hass]]"
+                domain="homeassistant"
+                service="reload_core_config"
+                >[[localize('ui.panel.config.server_control.section.reloading.core')]]
+              </ha-call-service-button>
+              <ha-call-service-button
+                hass="[[hass]]"
+                domain="group"
+                service="reload"
+                hidden$="[[!groupLoaded(hass)]]"
+                >[[localize('ui.panel.config.server_control.section.reloading.group')]]
+              </ha-call-service-button>
+              <ha-call-service-button
+                hass="[[hass]]"
+                domain="automation"
+                service="reload"
+                hidden$="[[!automationLoaded(hass)]]"
+                >[[localize('ui.panel.config.server_control.section.reloading.automation')]]
+              </ha-call-service-button>
+              <ha-call-service-button
+                hass="[[hass]]"
+                domain="script"
+                service="reload"
+                hidden$="[[!scriptLoaded(hass)]]"
+                >[[localize('ui.panel.config.server_control.section.reloading.script')]]
+              </ha-call-service-button>
+            </div>
+          </ha-card>
+        </template>
+        <ha-card
+          header="[[localize('ui.panel.config.server_control.section.server_management.heading')]]"
+        >
+          <div class="card-content">
+            [[localize('ui.panel.config.server_control.section.server_management.introduction')]]
+          </div>
+          <div class="card-actions warning">
+            <ha-call-service-button
+              class="warning"
+              hass="[[hass]]"
+              domain="homeassistant"
+              service="restart"
```

Comment:
Restarting and shutting down Home Assistant should not be hidden by an advanced flag. It should be accessible to all.